Jenkinsfile

```diff
@@ -7,20 +7,35 @@ node {
             script: 'make version',
             returnStdout: true
         ).trim()
+        def props = [ version: version ]
 
         stage("Testing ${version}") {
             sh 'make test-docker'
         }
         stage("Build image ${version}") {
             sh 'make build-docker'
         }
-
         if (env.BRANCH_NAME == "master") {
             stage("Push image") {
                 sh 'make push publish'
             }
+        } else {
+            def branchMatch = env.BRANCH_NAME =~ /^release-(0|[1-9]\d*)\.(0|[1-9]\d*)\.x$/
```

Comment:
why not just do `.*`?

Reply:
To capture the version and check the version matches (= we didn't forget to update the version file)